coalib/processes/Processing.py

```diff
@@ -587,6 +591,11 @@ def process_queues(processes,
     :return:                   Return True if all bears execute successfully and
                                Results were delivered to the user. Else False.
     """
+    debug_bears = (False
+                   if 'debug_bears' not in section or (
+                   section['debug_bears'].value == 'False') else (
+                   typed_list(str)(section['debug_bears'])))
```

Comment:
You are doing it too often imo. There should be one location to extract the actual `debug` setting from the section, and then pass it down via parameters.